### PR TITLE
Removed the handling of private packs in xsiam upload

### PR DESCRIPTION
**Pull request by moishce-zz**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-632

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Tests/Marketplace/upload_packs.py [hunk 1]
```diff
@@ -1068,9 +1068,15 @@ def main():
     diff_files_list = content_repo.commit(current_commit_hash).diff(content_repo.commit(previous_commit_hash))
 
     # taking care of private packs
-    is_private_content_updated, private_packs, updated_private_packs_ids = handle_private_content(
-        index_folder_path, private_bucket_name, extract_destination_path, storage_client, pack_names, storage_base_path
-    )
+    if marketplace == 'xsoar':
+        # taking care of private packs
```
Comment (ShahafBenYakir) [inline Tests/Marketplace/upload_packs.py:1072]:
```suggestion
```

**Review by ShahafBenYakir — COMMENTED** [review]
@moishce What about removing the logic from the cleanup function?
@Noy-Maimon anything else we might have missed? This looks a bit too simple.

**Review by ShahafBenYakir — CHANGES_REQUESTED** [review]
@moishce What about removing the logic from the cleanup function?
@Noy-Maimon anything else we might have missed? This looks a bit too simple.

**Comment by moishce** [comment]
@Noy-Maimon [a valid test upload](https://code.pan.run/xsoar/content/-/pipelines/3093058)

**Comment by ShahafBenYakir** [comment]
@Noy-Maimon @moishce what about this?
>@moishce What about removing the logic from the cleanup function?

**Comment by moishce** [comment]
@ShahafBenYakir on the cleanup function it’s already implemented

**Review by ShahafBenYakir — APPROVED** [review]

**Review by Noy-Maimon — APPROVED** [review]

**Pull request closed without merging**